src/raglite/_config.py

```diff
@@ -53,6 +67,12 @@ class RAGLiteConfig:
         ),
         compare=False,  # Exclude the reranker from comparison to avoid lru_cache misses.
     )
+    search_method: "SearchMethod" = field(default_factory=_default_search_method, compare=False)
+    system_prompt: str | None = None
+    rag_instruction_template: str = RAG_INSTRUCTION_TEMPLATE
+    num_chunks: int = 5
+    chunk_neighbors: tuple[int, ...] | None = (-1, 1)  # Neighbors to include in the context.
+    reranker_oversample: int = 4  # How many extra chunks to retrieve for reranking (multiplied).
```

Comment:
This would be subsumed by the `retrieval: ChunkSpanSearchMethod = partial ` proposal above.